website/docs/language/modules/develop/providers.mdx

```diff
@@ -102,7 +102,7 @@ features are needed by other parts of their overall configuration.
 ## Implicit Provider Inheritance
 
 For convenience in simple configurations, a child module automatically inherits
-default (un-aliased) provider configurations from its parent. This means that
+[default](/language/providers/configuration#default-provider-configurations) (un-aliased/ omitted `provider` block) provider configurations from its parent. This means that
```

Comment:
I like this! The parentheses are a bit awkward in this sentence, so now that you've added this nice like, I'd like to suggest taking that out and letting folks click the link to learn more about this concept if they need to.

Thoughts?

```suggestion
[default provider configurations](/language/providers/configuration#default-provider-configurations) from its parent. This means that
```

Reply:
Many thanks @laurapacilio, this makes sense!